### ISSUE TITLE
Pass SSO labels to workflows from conworkflows.

### ISSUE THREAD

**Issue by nityanandagohain** [argoproj/argo-workflows#7286]
# Summary

When argo server is running in SSO mode, and user creates a cronworkflow labels about the user is added to the resource. eg:- `creator-email` , `creator` .

Now when a cronworkflow is triggered this labels should be propagated to the workflow.

# Use Cases

When would you use this?
We have logic in our workflows where we utilize the labels to identify the user through which this workflow is running. In this case user created a cronworkflow which triggered this workflow.

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

**Comment by louisnow** [comment]
Runtime generated workflows also don't get the `creator-email` , `creator`, etc labels attached.

**Comment by nityanandagohain** [comment]
If I try to pass the labels using workflow metadata
eg:- 
```
metadata:
  name: test-latest
  namespace: default
  labels:
    workflows.argoproj.io/creator: 16507310-5346-41d6-b465-569eb14abcfe
    workflows.argoproj.io/creator-email: nityananda.at.atlan.com
spec:
  workflowSpec:
    templates:
      - name: whalesay
        inputs: {}
        outputs: {}
        metadata: {}
        container:
          name: ''
          image: 'alpine:3.6'
          command:
            - sh
            - '-c'
          args:
            - date; sleep 90
          resources: {}
    entrypoint: whalesay
    arguments: {}
  schedule: '* * * * *'
  concurrencyPolicy: Replace
  startingDeadlineSeconds: 0
  workflowMetadata:
    creationTimestamp: null
    labels:
      workflows.argoproj.io/creator: '{{workflows.argoproj.io/creator}}'
```


It throws this error.

<img width="1281" alt="image" src="https://user-images.githubusercontent.com/26831659/143845338-3947face-be75-40c9-b6d8-0f2ac461cf58.png">

**Comment by louisnow** [comment]
`creator` label is also not passed during a resubmit of a workflow.
`creator-email` however is passed during a resubmit

**Comment by umi0410** [comment]
🎉 This has been fixed by #11407 in a simple way. From now on, you won't need to set the creator labels of a child Workflow.

If a CronWorkflow has creator labels, they will be automatically propagated to a child Workflow!
